<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>581</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Files</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Input File Path</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_input">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>341</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_input">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>30</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Select</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Output Directory</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_output">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>70</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Choose</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_output">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>341</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>581</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>File Type</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_input_type">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Image</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Video</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_detector">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>30</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>YOLO</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>30</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Detector</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_detector">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>110</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Choose</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_detector">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>341</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Detector Directory</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Confidence</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_confidence">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.050000000000000</double>
     </property>
     <property name="value">
      <double>0.500000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>60</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Threshold</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_threshold">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>60</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.050000000000000</double>
     </property>
     <property name="value">
      <double>0.300000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>581</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Detection</string>
    </property>
    <widget class="QPushButton" name="pushButton_start">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>30</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Process Status</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_process">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>341</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 127);</string>
     </property>
     <property name="text">
      <string>[ IDLE ] Press Start to begin Processing the Input Data</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>comboBox_input_type</tabstop>
  <tabstop>comboBox_detector</tabstop>
  <tabstop>doubleSpinBox_confidence</tabstop>
  <tabstop>doubleSpinBox_threshold</tabstop>
  <tabstop>lineEdit_detector</tabstop>
  <tabstop>pushButton_detector</tabstop>
  <tabstop>lineEdit_input</tabstop>
  <tabstop>pushButton_input</tabstop>
  <tabstop>lineEdit_output</tabstop>
  <tabstop>pushButton_output</tabstop>
  <tabstop>lineEdit_process</tabstop>
  <tabstop>pushButton_start</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
